src/less/blog-v2/main.less

```diff
@@ -138,14 +138,54 @@ body {
       text-align: center;
       width: 300px;
     }
+  }  
+}
+.join-bar-article {
+  .font-poppins-bold;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: linear-gradient(120deg,#ffcc45,#4023fd);
+  width: 100%;
+  padding: @margin-md;
+  .join-message-article {
+    margin: @margin-xs 0;
+    text-align: center;
+    color: #f2f2f2;
+    display: inline-block;
+    .font-poppins;
+    font-size: 30px;
+  }
 
-    .join-message {
-    }
+  img {
+    width:300px;
+    height:auto;
+    padding-bottom:20px;
   }
 
-  @media (min-width: @screen-md-min) {
-    .signup-button {
-    }
+  .join-message-description {
+  text-align:center;
```

Comment:
Not required to update right now, but just watch out for the tabbing and spacing next time. Also I'd recommend adding the mixins first, followed by the individual css properties. No technical reason, just a readability thing. 

```less
.join-message-description {
  .font-poppins;
  text-align: center;
  color: #f2f2f2;
  font-size: 18px;
}
```